C#/cs01/tests/11.cs

```diff
@@ -0,0 +1,42 @@
+using NUnit.Framework;
+using Brainchild.Curriculum.Solutions;
+namespace Brainchild.Curriculum
+{
+    [TestFixture]
+    public class Test11
+    {
+        [SetUp]
+        public void SetUp()
+        {
+        }
+        
+        [Test]
+        public void CheckingFullNaming()
+        {
+            Sreerag sreerag = new Sreerag("Sreerag P S");
+          /*  sreerag.FirstName ="Sreerag";
+            sreerag.MiddleName = "P";
+            sreerag.LastName = "S";*/
+            string expecting = "Sreerag P S";
+            Assert.AreEqual(sreerag.FullName,expecting);
+        }
+        [Test]
+        public void CheckingWithoutMiddleNaming()
+        {
+            Sreerag sreerag = new Sreerag("Sreerag Ps");
+           /* sreerag.FirstName = "Sreerag";
```

Comment:
why did you comment out old code ? your code should be a new function without affecting existing function and test